keyboards/xd75/keymaps/hybrid/keymap.c

```diff
@@ -0,0 +1,481 @@
+/* Copyright 2020 Hybrid
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Layer shorthand
+#define _MAIN 0
+#define _LY1 1
+#define _LY2 2
+
+
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+  QMKBEST = SAFE_RANGE,
+  QMKURL
+};
+
+enum {
+  TD_A = 0,
+  TD_E,
+  TD_I,
+  TD_O,
+  TD_U,
+  TD_SLASH,
+  TD_UNDER,
+  TD_OP_ACCOL,
+  TD_CL_ACCOL,
+  TD_QUESTION,
+  TD_CEDILLE,
+  TD_DOT,
+};
+
+
+
+void dance_a_accent (qk_tap_dance_state_t *state, void *user_data) {
+  if (state->count == 2) {
+    register_code (KC_NLCK);
+    unregister_code (KC_NLCK);
```

Comment:
I wont add suggestions for all, but you can do
```suggestion
    tap_code (KC_NLCK);
```

Reply:
In fact this entire array could be replaced with the `send unicode` functions. 

https://docs.qmk.fm/#/feature_unicode?id=send_unicode_string

Reply:
I can see in 7761dcbfd2988f14ffe24d8b7350c9fa9badf226 you have updated one of the instances, but the rest really need to be addressed too.

Reply:
I reviewed the code with your comments & Drashna. 
I fixed all except the tap dance functions. I cannot test the send_unicode_string right now but I will.
Thanks for the tap_code() function :)